.gitignore

```
@@ -29,4 +29,5 @@ cabal.sandbox.config
 *.swp
 *.bak
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode/tasks.json
```

`.vscode/tasks.json`を追加するのは良い変更ですが、特定のエディタ設定を無視する場合は`.vscode/`ディレクトリ全体を無視する方が一般的です。

```suggestion
.vscode/
```